lib/util/cache/repository/impl/base.ts

```diff
@@ -76,30 +50,12 @@ export abstract class RepoCacheBase implements RepoCache {
       }
       const oldCache = JSON.parse(rawOldCache) as unknown;
 
-      if (isValidRev13(oldCache)) {
-        await this.restoreFromRev13(oldCache);
+      if (schema.match(RepoCacheV13, oldCache)) {
```

Comment:
@viceice @Gabriel-Ladzaretti This is our runtime schema validation with type narrowing